src/tasks/test-deployment.ts

```diff
@@ -31,12 +31,19 @@ const OUTPUT_FOLDER = "./output";
 
 const defaultTokens = {
   usdc: {
+    "1": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+    "100": "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83",
     "4": "0x4DBCdF9B62e891a7cec5A2568C3F4FAF9E8Abe2b",
   },
   weth: {
```

Comment:
@fedgiac 
here, we could have used the name nativeToken. Maybe we can adapt it in a follow up PR

Reply:
Agreed. :+1: